packages/playground/src/consts/url.ts

```diff
@@ -1,2 +1,3 @@
 export const baseUrl = `http://${String(process.env.HOST)}:${String(process.env.PORT)}`;
-export const oidcDomain = process.env.RAZZLE_OIDC_BASE_DOMAIN ?? 'N/A';
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+export const oidcDomain: string = process.env.RAZZLE_OIDC_BASE_DOMAIN ?? 'N/A';
```

Comment:
why is this unsafe?

Reply:
Because we don't have type defined for env variables at this point. So the type of `process.env.RAZZLE_OIDC_BASE_DOMAIN` is `any`, and you cannot assign `any` to another variable, thus the `unsafe-assignment`.
We will have to either use `.env` files or include `@types/node` in order to infer env variable types, I think.

Reply:
I somehow find it easier to solve this lint issue by simply wrapping it inside a `String()`... Please review